hass_nabucasa/__init__.py

```diff
@@ -263,12 +268,11 @@ def load_config():
         self.access_token = info["access_token"]
         self.refresh_token = info["refresh_token"]
 
-        orig_token = self.id_token
-
         await self.auth.async_check_token()
 
         # A refresh will trigger a start, so only start if we didn't refresh
```

Comment:
This needs fixing. We should always await start during initialize. Now if it did refresh it will start in a new task.